File: 1-Iniciante/Ex1011.java
import java.util.Scanner;

public class Ex1011 { //para ser aceito no Beecrowd mude o nome da classe para Main

    public static void main(String[] args) {
        // variaveis & objetos
        Scanner scanner = new Scanner(System.in);
        Double volume;
        final Double PI = 3.14159;

        // entrada
        Double raio = scanner.nextDouble();

        // processamento
        volume = (4.0 / 3) * PI * Math.pow(raio, 3);

        // saida
        System.out.printf("VOLUME = %.3f\n", volume);
        scanner.close();
    }
}

File: 1-Iniciante/Ex1011.kt
import java.util.Scanner
import kotlin.math.pow

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)

    val raio = scanner.nextDouble()

    val PI = 3.14159
    val volume = (4.0/3) * PI * raio.pow(3)

    println("VOLUME = ${String.format("%.3f", volume)}")
}
